msgpack/include/object.hpp

```diff
@@ -0,0 +1,20 @@
+#ifndef ARK_MSGPACK_OBJ_HPP
+#define ARK_MSGPACK_OBJ_HPP
+
+#include <variant>
+#include <iostream>
+#include <sstream>
+#include <msgpack.hpp>
+#include <Ark/Module.hpp>
+
+using CObject = std::variant<bool, double, std::string, std::vector<Value>>;
+extern CObject get_cobject(const Value &ark_object, ValueType type);
+namespace ArkMsgpack
+{
+	extern Value msgpack_obj_handle(std::vector<Value> &args);
+	extern Value msgpack_sbuffer(std::vector<Value> &args);
+	extern Value msgpack_obj(std::vector<Value> &args);
+	extern Value print_msgpack_o(std::vector<Value> &args);
```

Comment:
don't use extern ; just write Value function(arguments).

the only function which needs to be exported is the mapping one

also, the functions now need to take an extra argument: `Ark::VM* vm`